src/duct/middleware/web.clj

```diff
@@ -124,4 +124,8 @@
     b))
 
 (defmethod ig/init-key ::format [_ options]
-  #(mm/wrap-format % (deep-merge mc/default-options options)))
+  (let [malformed-handler (or (:malformed-handler options)
+                              (ig/ref :duct.handler.static/bad-request-malformed))]
```

Comment:
Supplying a reference here won't work. In the current alpha of Duct there is `prep-key` for such eventualities, but in the current stable version it needs to be hardcoded or set by a module.

Reply:
I know, I 've created the PR from 52cd93ca21e43710b0d725ed957dadb510a026b6

Reply:
OK I rebased from master. Could you suggest how to solve it without `ig/ref`?